flow/packet-headers/ipv4.yaml

```diff
@@ -112,15 +112,21 @@ components:
               Source address
             format: ipv4
```

Comment:
Since this PR is for traffic over DHCPv4 interface only, we should change the title accordingly and mention that separate PR will be created for DHCPv6 with similar method as we converge to for DHCPv4 in this PR.

Reply:
made the necessary changes